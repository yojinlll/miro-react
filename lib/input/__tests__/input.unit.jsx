import * as renderer from 'react-test-renderer'
import React from 'react'
import Input from '../input'

describe('input', ()=>{
  it('存在', ()=>{
    const json = renderer.create(<Input />).toJSON()
    expect(json).toMatchSnapshot()
  })
})