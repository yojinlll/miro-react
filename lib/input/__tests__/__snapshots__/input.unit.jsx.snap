// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input 存在 1`] = `
<input
  className="miro-input"
/>
`;
